pkg/testutil/bq.go

```diff
@@ -0,0 +1,89 @@
+// Copyright 2023 The Authors (see AUTHORS file)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package testutil provides functions to query bigquery table and upload GCS objects.
+package testutil
```

Comment:
https://dave.cheney.net/2019/01/08/avoid-package-names-like-base-util-or-common

Reply:
I don't blame @sailorlqh for the naming since we had "bad" examples. Any suggestion? Does the name matter if keep it internal?

Reply:
Since this is all about BigQuery, maybe `bqhelpers` or `bqtest`?